terra-jupyter-python/smoke_test.py

```diff
@@ -0,0 +1,44 @@
+"""Brief smoke tests for terra-jupyter-python.
```

Comment:
Is this executed by the GitHub action or just the `ipynb` notebooks?

Reply:
The GitHub Action https://github.com/deflaux/terra-docker/blob/add-smoke-tests/.github/workflows/test-terra-jupyter-python.yml#L94

Reply:
Ah I see